cmd/scheduled-feed/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/ossf/package-feeds/feeds/scheduler"
+
+	log "github.com/sirupsen/logrus"
+
+	"gocloud.dev/pubsub"
+	_ "gocloud.dev/pubsub/gcppubsub"
+)
+
+var delta = 5 * time.Minute
+
+func Poll(w http.ResponseWriter, r *http.Request) {
+	topicURL := os.Getenv("OSSMALWARE_TOPIC_URL")
```

Comment:
Check for `OSSMALWARE_TOPIC_URL` being empty should be handled in `main` with a panic,